examples/dataflow-flex-python/df-sample-xml-to-bq/xmltobq.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import logging
+
+import apache_beam as beam
+import apache_beam.io.fileio as fileio
+
+table_schema = {
```

Comment:
The BigQuery table schema does not match the XML schema at https://learn.microsoft.com/en-gb/dotnet/standard/linq/sample-xsd-file-customers-orders.

Reply:
Scema generated using xsdata and its matching BQ schema now.